nrfx/haly/nrfy_lpcomp.h

```diff
@@ -169,6 +169,23 @@ NRFY_STATIC_INLINE bool nrfy_lpcomp_sample_check(NRF_LPCOMP_Type * p_reg)
     return sample;
 }
 
+/**
+ * @brief Function for reading the current state of the LPCOMP.
+ *
+ * @param[in] p_reg Pointer to the structure of registers of the peripheral.
+ *
+ * @retval 0 The input voltage is below the threshold.
+ * @retval 1 The input voltage is above the threshold.
+ */
+NRFY_STATIC_INLINE uint32_t nrfy_lpcomp_sample(NRF_LPCOMP_Type * p_reg)
```

Comment:
Isn't it the same as `nrfy_lpcomp_sample_check()`?

Reply:
Exactly it is, see https://github.com/nrfconnect/nrfx/pull/659#discussion_r1738229328